examples/README.txt

```diff
@@ -0,0 +1,4 @@
+Examples
+========
+
+Below is a gallery of example of metric-learn use cases.
```

Comment:
...a gallery of example metric-learn use cases.

Reply:
Thanks, done